C#/Botkit/Microsoft.BotKit.Adapters.Slack/SlackAdapter.cs

```diff
@@ -346,7 +362,7 @@ public async void ProcessActivityAsync(HttpRequestMessage request, HttpResponseM
                 return;
             }
 
-            if (!VerifySignatureAsync(request, response))
+            if (!VerifySignature(request, response))
```

Comment:
This `if` is confusing at first glance, I think that this validation is missing something that it is not implemented in the original code at the moment.
However, do we really need an empty `if` clause? Can't it be rewritten more clearly?